lib/rav_codebreaker/console.rb

```diff
@@ -0,0 +1,123 @@
+require 'colorize'
+module RavCodebreaker
+  class Console
+    SCORES_FILE_NAME = './score.dat'
+
+    def initialize(level = :expert)
+      @score = []
+      @game = Game.new(level)
+    end
+
+    def play
+      @game.start
+      welcome_message
+      loop do
+        puts @game.format_error? ? incorrect_message : invitation_message
+        get_offer
+        exit if @game.exit?
+        show_hint if @game.show_hint?
+        next if @game.show_hint? || @game.format_error?
+        show_result_message
+        show_winner_message if win?
+        @game.next_turn 
+        show_gameover_message if @game.game_over?
+        break if win? || @game.game_over?
+      end
+      load_scores_from_file
+      save_results
+      show_results
+    end
```

Comment:
1. make method shorter
2. move logic to game
3. try to avoid ifs or use case instead